### PR TITLE
SISRP-15118, advising student profile card with photo_unavailable_bear

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.berkeley.edu/browse/SISRP-15118

Advising-student-overview page is done except for photo on student profile card.
* `fixFormattedAddresses` util is used in three different spots so I made it a reusable function in profileService

### CODE REVIEW

File: src/assets/templates/widgets/advisor/student_profile.html [hunk 1]
```diff
@@ -2,23 +2,81 @@
...
   </div>
-  <div class="cc-widget-text" data-cc-spinner-directive="academics.isLoading">
-
-    <div data-ng-bind="student.attributes.defaultName"></div>
+  <div class="cc-widget-text" data-cc-spinner-directive="academics.profile.isLoading">
+    <div class="cc-table cc-table-top-border cc-clearfix">
+      <table>
+        <tbody>
+        <tr>
+          <th width="92px">
```
Comment (raydavis) [inline src/assets/templates/widgets/advisor/student_profile.html]: This width seems like a natural for CSS, especially given its repeated appearance.

**Comment by raydavis** [comment]
Javascript side looks good; just had the inline style question.

File: src/assets/templates/widgets/advisor/student_profile.html (hunk 1 above)
Comment (johncrossman) [inline src/assets/templates/widgets/advisor/student_profile.html]: I agree, @raydavis  Amendment coming soon.

**Comment by johncrossman** [comment]
@raydavis @christianvuerings  One green Travis.

File: src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js [hunk 2]
```diff
@@ -1,31 +1,14 @@
...
-        element.address.formattedAddress = apiService.profile.fixFormattedAddress(formattedAddress);
-      }
-      return element;
-    });
-  };
-
   var parsePerson = function(data) {
     apiService.profile.parseSection($scope, data, 'emergencyContacts');
-    fixFormattedAddresses();
+    $scope.items.content = apiService.profile.fixFormattedAddresses($scope.items.content);
```
Comment (christianvuerings) [inline src/assets/javascripts/angular/controllers/widgets/profile/emergencyContactController.js]: @johncrossman thank you for extracting that!

File: src/assets/templates/widgets/advisor/student_profile.html [hunk 3]
```diff
@@ -1,24 +1,83 @@
...
-  <div class="cc-widget-text" data-cc-spinner-directive="academics.isLoading">
-
-    <div data-ng-bind="student.attributes.defaultName"></div>
+  <div class="cc-widget-text" data-cc-spinner-directive="profile.isLoading">
+    <div class="cc-table cc-table-top-border cc-clearfix">
+      <table class="cc-widget-profile-table">
+        <tbody>
+        <tr>
+          <th>
+            <div alt="{{profile.defaultName}}'s Profile Picture" data-ng-include="'widgets/delegate_student_img.html'"></div>
```
Comment (christianvuerings) [inline src/assets/templates/widgets/advisor/student_profile.html]: @johncrossman we should not have an `alt` attribute on the `div`, could we remove it?

Comment (johncrossman) [inline src/assets/templates/widgets/advisor/student_profile.html]: Follow up PR will have that removed, @christianvuerings

**Pull request merged**